### PR TITLE
[WIP] Add kickstart tar image installation test

### PR DESCRIPTION
**Pull request by jikortus**
Related: rhbz#1733504

This is (still) in WIP state, might need some more modifications.

### CODE REVIEW

File: test/run [hunk 1]
```diff
@@ -9,6 +9,8 @@ if [ -n "$TEST_SCENARIO" ]; then
     test/check-cli TestLiveIso
   elif [ "$TEST_SCENARIO" == "qcow2" ]; then
     test/check-cli TestQcow2
+  elif [ "$TEST_SCENARIO" == "tar" ]; then
+    test/check-cli TestTar
```
Comment (atodorov) [inline test/run]: Add `rhel-7-7/tar` as manual in testmap.py and kick off this scenario manually to see if it works

File: tests/cli/test_compose_tar_kickstart.sh [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        uuid=$(echo $uuid | cut -f 2 -d' ')
+    rlPhaseEnd
+
+    rlPhaseStartTest "compose finished"
+        if [ -n "$uuid" ]; then
+            until $CLI compose status | grep -E "$uuid (FINISHED|FAILED)"; do
+                sleep 60
+                rlLogInfo "Waiting for compose to finish ..."
+            done;
+            #check_compose_status "$UUID"
```
Comment (atodorov) [inline tests/cli/test_compose_tar_kickstart.sh]: Remove comment ?

Comment (jikortus) [reply tests/cli/test_compose_tar_kickstart.sh]: Thanks for spotting this.

File: test/run (hunk 1 above)
Comment (jikortus) [reply test/run]: Does this mean only adding `rhel-7-7/tar` into `testmap.py` in cockpit/bots project? I'm not really sure what determines the manual/automatic execution in this case.

Comment (atodorov) [reply test/run]: When you open the testmaps.py file you will understand. There is a manual section and there are sections that get triggered automatically.

Comment (atodorov) [reply test/run]: Use `_manual` key in the dict contained in this file. Search for it to see examples.

Comment (jikortus) [reply test/run]: Thank you for the hint. I hope I understood that correctly: https://github.com/cockpit-project/bots/pull/86

**Pull request merged**